src/main/java/com/aws/greengrass/shadowmanager/model/configuration/ShadowSyncConfiguration.java

```diff
@@ -225,37 +226,37 @@ private static void processThingShadowSyncConfiguration(Set<ThingShadowSyncConfi
         Validator.validateThingName(thingName);
 
         ThingShadowSyncConfiguration syncConfiguration;
-        boolean syncClassicTopic = true;
-        for (Map.Entry<String, Object> configObjectEntry : thingConfig.entrySet()) {
-            switch (configObjectEntry.getKey()) {
-                case CONFIGURATION_CLASSIC_SHADOW_TOPIC:
```

Comment:
This switch statement was better than splitting it into multiple ifs. When reading the code this tells us there are only 2 possible keys that are accepted. The ifs do not reveal that intent.

Reply:
I left this untouched, thank you! :)